### PR TITLE
Update repositories.txt - Library name change for Alfredo_NoU3

### PR DESCRIPTION
**Pull request by BotSpace**
I'm the maintainer for this library. I am about to start distributing the board that this library is written for, and it is gonna bug me forever if this isn't fixed before the community that uses my board starts using this library. Thank you!

### CODE REVIEW

**Comment by BotSpace** [comment]
@ArduinoBot

**Review by per1234 — COMMENTED** [review]
Hello @BotSpace. Thanks for your caring about the accuracy of the Arduino Library registry data! I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by umbynos — APPROVED** [review]
Done in https://github.com/arduino/library-registry/pull/4955#pullrequestreview-2228076863

**Pull request merged**
